### PR TITLE
hide dev mode setting

### PR DESCRIPTION
**Pull request by flexsurfer**

### CODE REVIEW

File: src/status_im/ui/screens/advanced_settings/views.cljs [hunk 1]
```diff
@@ -39,69 +39,27 @@
...
-    :accessories
-    [[react/switch
-      {:track-color #js {:true colors/blue :false nil}
-       :value       dev-mode?
-       :on-value-change
-       #(re-frame/dispatch
-         [:multiaccounts.ui/dev-mode-switched (not dev-mode?)])
-       :disabled    false}]]}
-   {:type :divider}])
+   #_{:type                    :small
```
Comment (cammellos) [inline src/status_im/ui/screens/advanced_settings/views.cljs:42]: shall we just remove it? we can always bring it back from git history?

**Review by cammellos — APPROVED** [review]

**Review by rachelhamlin — APPROVED** [review]

**Comment by flexsurfer** [comment]
@churik could you take a look at tests results

**Comment by churik** [comment]
Needs to be changed, please wait for my commit

**Comment by churik** [comment]
@flexsurfer ready

**Pull request merged**